h5bench_patterns/README.md

```diff
@@ -0,0 +1,37 @@
+## Additions to H5Bench_read
+4 new patterns are added to the h5bench read benchmark:
```

Comment:
Make sure to mention that this works only with single MPI process.

Reply:
Added.